fastapi_oidc/auth.py

```diff
@@ -83,7 +83,9 @@ def get_auth(
     #    doesn't support OpenIDConnect yet.
     #
     # Some relevant examples https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/
-    oauth2_scheme = OpenIdConnect(openIdConnectUrl=issuer)
```

Comment:
Nice! We can delete the comments above since their now obsolete.

Reply:
Sure, I deleted them in the other PR